guide/interactions/modals.md

```diff
@@ -0,0 +1,117 @@
+# Modals
+
+With modal interactions you can create pop-out forms that allow users to provide you with formatted input. We'll cover how to create, send and receive modals using discord.js!
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) page](/interactions/slash-commands.md). Please carefully read those first so that you can understand the methods used in this section.
+:::
+
+# Building and Responding with Modals
+
+Unlike message components, modals aren't strictly components themselves. They're a callback structure used to respond to interactions.
+
+::: tip
+You can have a maximum of five `ActionRow`s per modal, and one `TextInputComponent` within an `ActionRow`. Currently, you cannot use `SelectMenuComponent`s or `ButtonComponents` in modal action rows.
+:::
+
+To create a modal you construct a new `Modal`. You can then use the setters to add the title.
+
+```js {1,7-13}
+const { Modal } = require('discord.js');
+
+client.on('interactionCreate', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'ping') {
+		const modal = new Modal()
+			.setCustomId('myModal')
+			.setTitle('My Modal');
+
+		// TODO: Add components to modal...
+
+		await interaction.showModal(modal);
+	}
+});
+```
+::: tip
+The custom ID is a developer-defined string of up to 100 characters.
```

Comment:
I'm pretty sure we should be lowercasing it. I've noticed some other places in the guide where "ID" is used, but that should probably be changed too?

```suggestion
The custom id is a developer-defined string of up to 100 characters.
```